### PR TITLE
NamedTemp manual deletion for windows users.

### PR DESCRIPTION
**Pull request by LotannaEzenwa**
Solution to the problem on PR #1282

### CODE REVIEW

**Comment by LotannaEzenwa** [comment]
@llllllllll @richafrank, Could we go over this tomorrow?

File: zipline/utils/cache.py [hunk 1]
```diff
@@ -41,7 +41,6 @@ class CachedObject(namedtuple("_CachedObject", "value expires")):
     >>> obj.unwrap(expires)
     1
     >>> obj.unwrap(expires + Timedelta('1 minute'))
-    ... # doctest: +IGNORE_EXCEPTION_DETAIL
```
Comment (llllllllll) [inline zipline/utils/cache.py]: We still need this line, this is what is causing the tests to fail in py3

File: zipline/utils/cache.py [hunk 2]
```diff
@@ -302,6 +304,7 @@ def __exit__(self, *exc_info):
         if exc_info[0] is None:
             self._commit()
         self._tmpfile.__exit__(*exc_info)
+        os.remove(self._tmpfile.name)
```
Comment (llllllllll) [inline zipline/utils/cache.py]: won't this try to remove twice, once in the commit and then here?

Comment (LotannaEzenwa) [inline zipline/utils/cache.py]: Yeah, that's true. Changing it.

**Pull request closed without merging**